src/radial-chart/index.js

```diff
@@ -270,18 +237,28 @@ class RadialChart extends React.Component {
             className="rv-radial-chart__series--pie"
             transform={`translate(${width / 2},${height / 2})`}
             ref="container">
-            {data.map((d, i) => <path {...{
-              d: arc(pieData[i]),
-              style: {
-                opacity: opacityFunctor && opacityFunctor(d),
-                stroke: strokeFunctor && strokeFunctor(d),
-                fill: fillFunctor && fillFunctor(d)
-              },
-              onMouseOver: e => this._sectionMouseOver(pieData[i], e),
-              onMouseOut: e => this._sectionMouseOut(pieData[i], e),
-              onClick: e => this._sectionClick(pieData[i], e),
-              key: i
-            }}/>)}
+            {data.map((d, i) => {
+              return (
+                <path {...{
+                  d: arc(pieData[i]),
+                  style: {
+                    opacity: opacityFunctor && opacityFunctor(d),
+                    stroke: strokeFunctor && strokeFunctor(d),
+                    fill: fillFunctor && fillFunctor(d)
+                  },
+                  onMouseEnter: e => {
```

Comment:
true, but ill have to extend the max line width. truth be told i'd been waiting for an excuse to do so, so here we go